app/controllers/media_objects_controller.rb

```diff
@@ -430,11 +431,14 @@ def update_status
             else
               errors += ["#{media_object.title} (#{id}) (permission denied)."]
             end
+          end
+        rescue ActiveFedora::RecordInvalid
+          errors += ["#{media_object.id}, validation failed."]
```

Comment:
Can you use or include the error message from the exception itself? It lists all the fields that are missing so the user knows what to fix, otherwise it's unclear what needs to be done.